src/javascripts/search.js

```diff
@@ -0,0 +1,58 @@
+UK_Parliament.search = function () {
+  var
+    header = document.querySelector('header'),
+    logo = header.querySelector('.uk_parliament'),
+    searchComponent = header.querySelector('.search__global'),
+    input = header.querySelector('input'),
+    button = header.querySelector('button'),
+    searchToggle = header.querySelector('.search__global__toggle');
+
+  if(searchComponent) {
+    checkScreenSize();
+    searchToggle.addEventListener('click', toggleSearch);
+    window.addEventListener('orientationchange', checkScreenSize);
+  }
+
+  function checkScreenSize() {
+    if (window.innerWidth <= 768) {
```

Comment:
should value '768' be stored as a variable?

Reply:
yep adding